dhis-2/dhis-api/src/main/java/org/hisp/dhis/scheduling/JobConfiguration.java

```diff
@@ -79,13 +83,23 @@
     // Externally configurable properties
     // -------------------------------------------------------------------------
 
-    private String cronExpression;
-
     /**
      * The type of job.
      */
     private JobType jobType;
 
+    /**
+     * The cron expression used for scheduling the job. Relevant for scheduling
+     * type {@link SchedulingType#CRON}.
+     */
+    private String cronExpression;
+
+    /**
+     * The delay in seconds between the completion of one job execution and the
+     * start of the next. Relevant for scheduling type {@link SchedulingType#FIXED_DELAY}.
+     */
+    private Integer delay;
```

Comment:
We also have a property in JobConfiguraition: "continuousExecution", which right now just reschedules the job after it finishes. I guess this PR removes the need for that property?

Reply:
Yes correct. I will create a new PR for removal of that property once this is merged.